addons/google_recaptcha/models/ir_http.py

```diff
@@ -24,7 +25,13 @@ def get_frontend_session_info(self):
 
     @api.model
     def _add_public_key_to_session_info(self, session_info):
-        """Add the ReCaptcha public key to the given session_info object"""
+        """Add the ReCaptcha configuration to the given session_info object"""
+        session_info['recaptcha_enabled'] = str2bool(
+            request.env['ir.config_parameter'].sudo().get_param(
+                'enable_recaptcha',
+                default=True
+            )
+        )
         public_key = self.env['ir.config_parameter'].sudo().get_param('recaptcha_public_key')
         if public_key:
             session_info['recaptcha_public_key'] = public_key
```

Comment:
I would avoid call sudo() twice here.
```suggestion
        """Add the ReCaptcha configuration to the given session_info object"""
        config_params = request.env['ir.config_parameter'].sudo()
        session_info['recaptcha_enabled'] = str2bool(
            config_params.get_param('enable_recaptcha', default=True)
        )
        public_key = config_params.get_param('recaptcha_public_key')
        session_info['recaptcha_public_key'] = public_key or ''
```